recipes/erlang/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+./configure --with-ssl="${PREFIX}" --prefix="${PREFIX}"
```

Comment:
Actually we can be really lazy here and add this flag (`"--enable-m${ARCH}-build"`) to `./configure`.